etcdserver/api/cluster.go

```diff
@@ -25,6 +25,8 @@ import (
 type Cluster interface {
 	// ID returns the cluster ID
 	ID() types.ID
+	// LocalID returns the local server ID
+	LocalID() types.ID
```

Comment:
My impression is that this is also a v2 interface. If we need to add more APIs, we might want to create a new interface. See my later comments in etcdhttp/peer.go